benchmark/BDN.benchmark/Cluster/ClusterContext.cs

```diff
@@ -40,18 +27,18 @@ public void Dispose()
             server.Dispose();
         }
 
-        public void SetupSingleInstance(bool enableCluster = true)
+        public void SetupSingleInstance(bool disableSlotVerification = false)
         {
             var opt = new GarnetServerOptions
             {
                 QuietMode = true,
-                EnableCluster = enableCluster,
-                Port = Port,
+                EnableCluster = !disableSlotVerification,
+                Port = port,
                 CleanClusterConfig = true,
             };
```

Comment:
Can you add the following? It will make @darrenge life easier because in his CI benchmark, it fails to create the cluster config file due to permission issue. 
```suggestion
            };
            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
                opt.CheckpointDir = "/tmp";
```